sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/eventhubs/properties/AzureEventHubsProperties.java

```diff
@@ -194,7 +194,7 @@ public void setPrefetchCount(Integer prefetchCount) {
          * Map event position to use for each partition if a checkpoint for the partition does not exist in
          * CheckpointStore.
          */
-        private Map<String, StartPosition> initialPartitionEventPosition = new HashMap<>();
+        private final Map<String, StartPosition> initialPartitionEventPosition = new HashMap<>();
```

Comment:
why do we need to change this to final here

Reply:
Due to the map having been initialized, we can make this property final as well. Shall we do this change?